sources/packages/backend/workflow/test/2021-2022/fulltime-assessment/appeals/fulltime-assessment-2021-2022-PDPPD-status.e2e-spec.ts

```diff
@@ -0,0 +1,102 @@
+import { PROGRAM_YEAR } from "../../constants/program-year.constants";
+import {
+  createFakeConsolidatedFulltimeData,
+  executeFullTimeAssessmentForProgramYear,
+} from "../../../test-utils";
+
+describe(`E2E Test Workflow fulltime-assessment-${PROGRAM_YEAR}-PDPPD-status.`, () => {
+  it("Should be true when student selected PD/PPD status through the application without any appeal.", async () => {
+    // Arrange
+    const assessmentConsolidatedData =
+      createFakeConsolidatedFulltimeData(PROGRAM_YEAR);
+    assessmentConsolidatedData.studentDataApplicationPDPPDStatus = "yes";
+
+    // Act
+    const calculatedAssessment = await executeFullTimeAssessmentForProgramYear(
+      PROGRAM_YEAR,
+      assessmentConsolidatedData,
+    );
+
+    // Assert
+    expect(calculatedAssessment.variables.calculatedDataPDPPDStatus).toBe(true);
+  });
+
+  it(
+    "Should be true when student did not select PD/PPD status from the application but requested a change " +
+      "selecting 'Yes, I have disability and want this application to be assessed for disability funding.'",
+    async () => {
+      // Arrange
+      const assessmentConsolidatedData =
+        createFakeConsolidatedFulltimeData(PROGRAM_YEAR);
+      assessmentConsolidatedData.studentDataApplicationPDPPDStatus =
+        "noIDoNotHaveADisability";
+      assessmentConsolidatedData.appealsStudentDisabilityAppealData = {
+        studentNewPDPPDStatus: "yes",
+      };
+
+      // Act
+      const calculatedAssessment =
+        await executeFullTimeAssessmentForProgramYear(
+          PROGRAM_YEAR,
+          assessmentConsolidatedData,
+        );
+
+      // Assert
+      expect(calculatedAssessment.variables.calculatedDataPDPPDStatus).toBe(
+        true,
+      );
+    },
+  );
+
+  it(
+    "Should be false when student selected PD/PPD status from the application but requested a change " +
+      "selecting 'No, I want to access other funding types while I wait for a verification decision.'",
+    async () => {
+      // Arrange
+      const assessmentConsolidatedData =
+        createFakeConsolidatedFulltimeData(PROGRAM_YEAR);
+      assessmentConsolidatedData.studentDataApplicationPDPPDStatus = "yes";
+      assessmentConsolidatedData.appealsStudentDisabilityAppealData = {
+        studentNewPDPPDStatus: "noIWantToAccessOtherFundingTypes",
+      };
+
+      // Act
+      const calculatedAssessment =
+        await executeFullTimeAssessmentForProgramYear(
+          PROGRAM_YEAR,
+          assessmentConsolidatedData,
+        );
+
+      // Assert
+      expect(calculatedAssessment.variables.calculatedDataPDPPDStatus).toBe(
+        false,
+      );
+    },
+  );
+
+  it(
+    "Should be false when student selected PD/PPD status from the application but requested a change " +
+      "selecting 'No, I do not have a disability.'",
+    async () => {
+      // Arrange
+      const assessmentConsolidatedData =
+        createFakeConsolidatedFulltimeData(PROGRAM_YEAR);
```

Comment:
I would say we can use the last scenario in this way which could be more probably be realtime. 
![image](https://github.com/bcgov/SIMS/assets/54600590/5acfa4de-f0a6-4461-9bfc-113281c85677)

Reply:
I added that scenario.

Reply:
Thank you